docs/openapi.yaml

```diff
@@ -0,0 +1,1143 @@
+      parameters:
+        - in: path
+          name: robot_id
+          required: true
+          schema:
+            type: integer
+            format: int32
+            example: 21
+        - in: path
+          name: mission_id
+          required: true
+          schema:
+            type: integer
+            format: int32
+            example: 287
+      requestBody:
+        content:
+          application/json:
+            schema: {}
+            examples:
+              start:
+                $ref: "#/components/examples/start"
+
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StartResponse"
+          description: Mission succesfully started
+        "403":
+          description: Failed to start - Client is not authorized to start mission
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        "409":
+          description: Failed to start - Conflict with schedule of robot
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+  /robots/{robot_id}/stop:
+    post:
+      operationId: postStopRobot
+      summary: Stop robot
+      description: |
+        ### Overview
+        Stop the execution of the current active mission. 
+
+        If there is no active mission on robot, nothing happens.
+      tags:
+        - Robots
+      parameters:
+        - in: path
+          name: robot_id
+          required: true
+          schema:
+            type: integer
+            format: int32
+            example: 21
+      requestBody:
+        content:
+          application/json:
+            schema: {}
+            examples:
+              stop:
+                $ref: "#/components/examples/stop"
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PostResponse"
+          description: Robot succesfully stopped
+        "403":
+          description: Failed to stop - Client is not authorized to stop mission
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+
+  /missions:
+    get:
+      operationId: getMissions
+      summary: Lookup all available missions on asset
+      description: |
+        ### Overview
+        Lookup up all available missions on asset in the Echo mission planner
```

Comment:
Shall we also list the draft missions in some manner?

Reply:
Haven't thought about it. Depends on the overall workflow we want. I guess it would be natural that draft missions isn't shown outside of the echo mission planner.

Reply:
Agreed, this would probably become clear after the process with UX designer is competed